include/api.php

```diff
@@ -5973,6 +5973,8 @@ function api_friendica_notification_seen($type)
 		return api_format_data('result', $type, ['result' => "success"]);
 	} catch (NotFoundException $e) {
 		throw new BadRequestException('Invalid argument');
+	} catch (Exception $e) {
+		throw new InternalServerErrorException('Internal Server exception');
```

Comment:
If this happens, do we have the actual error (in `$e->getMessage()` logged anywhere?

Reply:
Now we have :) The origin exception will now get chained to the current exception and is now shown in the stack trace at `$e->getTraceAsString()`